upcloud_api/cloud_manager/base.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, token, timeout=None):  # noqa
         self.token = token
         self.timeout = timeout
 
-    def request(self, method, endpoint, body=None):
+    def request(self, method, endpoint, body=None, timeout=-1):
```

Comment:
I would prefer timeout to be None

Reply:
The point for the default _not_ to be `None` is that one can then call the method with `timeout=None` to disable the timeout altogether. (See complete commit message of the first commit.)

Reply:
ok, works for me :D, merging